.rubocop.yml

```diff
@@ -13,8 +15,21 @@ Metrics/LineLength:
   Description: 'Limit lines to 110 characters.'
   Max: 110
 
+Metrics/MethodLength:
+  Description: 'Limit methods to 15 lines.'
+  Max: 15
```

Comment:
I've almost never had an issue with it when my methods are only doing one thing (which should be the case)